### DISCUSSION TITLE
Glimmer (GLMR)

### DISCUSSION THREAD

**Discussion by ethvv** [ethvv/v#752] [Binance Research]
<h1 align="center">Glimmer (GLMR)</h1>
<p align="center"><em>一个基于Polkadot网络与EVM兼容的区块链。</em></p>
<p align="center"><em>2022.1.11</em></p>
<hr/>
<ul class="jsx-2167690016"><li><p class="jsx-376140997">Moonbeam是在Polkadot网络上作为一个平行链运行的一个与EVM<strong><a href="https://academy.binance.com/en/glossary/virtual-machine" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">兼容的区块链</a></strong>。与EVM兼容允许基于以太坊的应用程序在Moonbeam上部署。</p></li><li><p class="jsx-376140997">该网络为开发者提供的功能包括部署Solidity智能合约、兼容Web3的API、预言机数据和以太网络桥接器。</p></li><li><p class="jsx-376140997">GLMR是Moonbeam的功能型和治理代币，具有以下<strong><a href="https://moonbeam.foundation/glimmer-token/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">用例</a></strong>：</p><ul class="jsx-2167690016"><li><strong>交易费用</strong>：用户可以用GLMR支付<strong><a href="https://academy.binance.com/en/glossary/gas" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">交易费</a></strong>和执行智能合约。</li><li><strong>抵押</strong>：用户可以抵押GLMR成为验证人或进行委托，参与网络共识，并得到GLMR的奖励。</li><li><strong>治理</strong>：GLMR代币持有者能够提名理事会成员并参与网络上的投票。</li><li><strong>流动性挖矿</strong>：为建立在Moonbeam区块链上的去中心化交易所（DEX）<strong><a href="https://academy.binance.com/en/articles/what-are-liquidity-pools-in-defi" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">提供流动性</a></strong>，以赚取部分交易所的费用。</li></ul></li><li><p class="jsx-376140997">Moonbeam由以下主要部分组成：</p><ul class="jsx-2167690016"><li><strong><a href="https://research.binance.com/en/projects/polkadot" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">Polkadot</a></strong>：Polkadot旨在协助跨链交流与操作互通。Moonbeam是Kusama生态系统中<strong><a href="https://moonbeam.network/networks/moonriver/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">Moonriver (MOVR)</a></strong>的Polkadot生态系统版本。</li><li><strong><a href="https://polkadot.network/launch-parachains/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">平行链</a></strong>：平行链是独立的区块链（Shards），能够连接到Polkadot。</li><li><strong><a href="https://moonbeam.network/education/what-is-a-crowdloan/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">众筹贷款</a></strong>：它是项目为Polkadot网络上的一个平行链插槽进行筹款的方法。用户暂时锁定他们的DOT，以换取项目团队的奖励，通常是以项目的原生代币形式。</li><li><strong><a href="https://polkadot.network/auctions/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">插槽拍卖</a></strong>：要想在Polkadot上作为平行链运营的项目，必须在平行链拍卖中赢得一个插槽。Moonbeam通过众筹赢得了<strong><a href="https://moonbeam.foundation/news/moonbeam-wins-the-second-parachain-auction/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">第二次插槽拍卖的名额</a></strong>，从2021年12月17日开始，为期96周，的Polkadot平行链。</li></ul></li><li><p class="jsx-376140997">该项目在众筹贷款中从<strong>~200K的贡献者</strong>那里筹集了<strong>~36MM的DOT代币</strong>，用于Moonbeam平行链插槽拍卖。</p></li><li><p class="jsx-376140997">截至2022年1月11日，GLMR的总代币供应量为<strong>1,000,000,000</strong>。GLMR的最大代币供应量没有上限，每年的通货膨胀率为5%。</p></li><li><p class="jsx-376140997">在<strong><a href="https://moonbeam.foundation/glimmer-token/" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">此处</a></strong>了解有关GLMR代币分配的更多信息。</p></li></ul>

**Comment by luckyguygx** [comment]
邮件收到了
